README.md

```diff
@@ -1,142 +1,220 @@
-
-1. Extract  wso2ei-6.5.0.zip and go to the extracted directory/bin.
-2. Run integrator.sh or integrator.bat.
-3. Point your favourite browser to  <https://localhost:9443/carbon>
-4. Use the following username and password to login
-   username : admin
-   password : admin
-
-## Running other runtimes individually (Analytics, Broker, Business-Process)
-
-1. Extract wso2ei-6.5.0 and go to the extracted directory.
-2. Go to wso2ei-6.5.0/wso2 directory.
-3. Go to appropriate runtime directory (analytics/broker/business-process) /bin.
-4. Execute wso2server.sh or wso2server.bat.
-5. Access the url related to the required runtime. (For example, use <https://localhost:9445/carbon> for the business-process runtime.)
-
-## Known Issues
-
-All the open issues pertaining to WSO2 Enterprise Integrator are reported at the following location:
+![WSO2 Enterprise Integrator](gh-docs/images/wso2-integration-logo.png?raw=true)
+
+[![Build Status][badge-travis-image]][badge-travis-url]
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/wso2/product-ei/blob/master/LICENSE)
+[![Twitter](https://img.shields.io/twitter/follow/wso2.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=wso2)
+
+# WSO2 Enterprise Integrator
+
+WSO2 Enterprise Integrator is an open source, fast, cloud native and scalable integration solution which can be 
+positioned as the core of WSO2 Integration Agile Platform. It enables enterprise integration developers to build 
+sophisticated integration solutions, achieving digital agility. As a mature integration product since 2005,
+*(branded as WSO2 ESB at the time)*, it continues to be the most sophisticated and extensible enterprise integration 
+solutions out there in open source.
+
+Actively maintained, with commercial support from WSO2 Inc, WSO2 Enterprise Integrator is widely used in production at 
+companies around the globe starting from startups to fortune 500 companies in the fields of government, healthcare, 
+banking, education, communication, etc.
+
+[Installation](https://docs.wso2.com/display/EI640/Installation+Guide) | 
+[Documentation](https://docs.wso2.com/display/EI640/) | 
+[Mailing Lists](https://wso2.com/mail/) | 
+[Blog](https://wso2.com/blogs/thesource/) | 
+[Support](https://wso2.com/subscription) | 
+[Nightly Builds](https://wso2.org/jenkins/view/products/job/products/job/product-ei/)
+
+
+## Outline 
+
+- [**Why WSO2 Enterprise Integrator?**](#Why-WSO2-Enterprise-Integrator)
+- [**Features**](#Features)
+- [**Distributions**](#Distributions)
+- [**Installation and Run**](#Installation-and-Run)
+- [**Artifact Development**](#Artifact-Development)
+- [**Building from Source**](#Building-from-Source)
+- [**Enterprise Support**](#Enterprise-Support)
+- [**Licence**](#License)
+
+
+## Why WSO2 Enterprise Integrator
+
+WSO2 Enterprise Integrator is an open source, hybrid integration platform. It is comprised of profiles those address different
```

Comment:
```suggestion
WSO2 Enterprise Integrator is an open source, hybrid integration platform. It is comprised of profiles that address different
```

Reply:
@samgnaniah No, Micro integrator will explain in seperate repo

Reply:
As per @kasun04 , we should link it as the cloud native distribution of EI. Will link it accordingly.